cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -286,6 +285,12 @@ spec:
   types. It is recommended to use the capacity-optimized allocation strategy,
   which will automatically launch Spot Instances into the most available pools
   by looking at real-time capacity data and.
+* Each Auto Scaling Group should be composed of instance types that provide
```

Comment:
I think this can probably be combined with the bullet point beginning

```
Set LaunchTemplateOverrides to include the 'base' instance...
```
to provide a clearer single paragraph.